packages/amplify-util-mock/src/__e2e_v2__/searchable-transformer.e2e.test.ts

```diff
@@ -61,19 +124,393 @@ describe('@searchable transformer', () => {
   /**
    * Test queries below
    */
-  test('@searchable allows the mock server to run', async () => {
+
+  if (isWindowsPlatform()) {
```

Comment:
I have spent some time to mock the CLI core appropriately but am running into issues. Since this logic is localized to this test, can I take it up later? We're planning to support windows too in the future, so this will go away anyways.